doc/doxygen/tutorial/tutorial.h.in

```diff
@@ -703,7 +712,9 @@
  *       step-40,
  *       step-42,
  *       step-50,
- *       step-55
+ *       step-55,
+ *       step-60,
```

Comment:
why do you add step-60 here? As far as I can tell, step-60 does not use MPI.

Reply:
You are right. I was thinking of the version I have on my PC.